src/stravalib/strava_model.py

```diff
@@ -4,7 +4,7 @@
 from __future__ import annotations
```

Comment:
I think we will want to update the [other repo here](https://github.com/stravalib/strava_swagger2pydantic/blob/main/swagger2pydantic.py#L87) to support this. i'll leave this in draft until i actually do that and regenerate the model from that script

Reply:
this is done!